tests/test_chdir/remote.bzl

```diff
@@ -0,0 +1,14 @@
+def _test_chdir_remote_impl(ctx):
+  ctx.file("WORKSPACE", """workspace("test_chdir_remote")""")
+  ctx.file("BUILD.bazel", """""")
```

Comment:
This is a very long way to write the empty string :)

Reply:
It wasn't always empty